### ISSUE TITLE
[ENHANCEMENT] bastille copy, much like scp

### ISSUE THREAD

**Issue by PythonLinks** [BastilleBSD/bastille#552]
**Is your feature request related to a problem? Please describe.**
I have a produciton container.  I clone it, make changes, and then want to copy those changes back to the production container.  Today it was /root/.emacs.d/

Right now I have to type in some ridiculously long "bastille cp" command.  

**Describe the solution you'd like**

 I would like to be able to do 

bastille copy -r debug:/root/.emacs.d production:/root/.emacs.d 

where debug and production are two containers.  
Maybe even allow an ALL argument as the target. 

I also often want to copy things out of a container to the host system. 
Without the : it would assume the file or directory is in the host system.  

Basically just like scp but for containers. 

Also an alias would be nice 
bcp -r debug:/root/.emacs.d production:/root/.emacs.d 

**Describe alternatives you've considered**
I just type in the long commands.  I could probably write this as a shell script myself. 
It is a good first project for someone who wants to learn how bastille works. 

**Additional context**
If bcp works like scp, it would be easy for people to grok it.

**Comment by tschettervictor** [comment]
Can't you do `bastille copy all ...` or do you just want those two jails?
@PythonLinks

**Comment by tschettervictor** [comment]
Bastille already has cp (host to jail) and rcp (jail to host)

Have you tried this?

**Comment by tschettervictor** [comment]
@PythonLinks plesse test #805. It has what you are asking for.

**Comment by bmac2** [comment]
#805 implements this feature.  Closing this request.
